util/util_test.go

```diff
@@ -647,3 +647,15 @@ func TestClusterToObjectsMapper(t *testing.T) {
 		g.Expect(f.Map(handler.MapObject{Object: cluster})).To(ConsistOf(tc.output))
 	}
 }
+
+func TestOrdinalize(t *testing.T) {
+	g := NewWithT(t)
+
+	input := []int{0, 1, 2, 43, 4, 5, 106, 207, 1008, -109, -0}
+	expected := []string{"0th", "1st", "2nd", "43rd", "4th", "5th", "106th", "207th", "1008th", "-109th", "0th"}
+
+	g.Expect(len(input)).To(Equal(len(expected)))
+	for i, in := range input {
+		g.Expect(Ordinalize(in)).To(Equal(expected[i]))
+	}
```

Comment:
On potential improvement for this test case would be to use the testCases struct and then `t.Run` pattern used in a lot of these tests. The benefit of this is that all test cases will get run no matter if one fails, at the moment, I think this test would fail early as soon as one of the cases fails. Very minor though as this is such a small method/test case, feel free to ignore